diff --git a/CMakeLists.txt b/CMakeLists.txt
index d643b0f..549ef7a 100644
--- a/extern/datetime-fortran/CMakeLists.txt
+++ b/extern/datetime-fortran/CMakeLists.txt
@@ -26,7 +26,7 @@ endif()

 # MinGW GCC needs this for strftime to work
 # https://sourceforge.net/p/mingw-w64/bugs/793/
-target_link_libraries(datetime PRIVATE "$<$<BOOL:${MINGW}>:ucrtbase>")
+target_link_libraries(datetime PRIVATE "$<$<BOOL:${MINGW}>:stdc++>")

 # tests
 if(BUILD_TESTING)
